packages/repository/test/acceptance/repository.acceptance.ts

```diff
@@ -71,6 +71,34 @@ describe('Repository in Thinking in LoopBack', () => {
     expect(stored).to.containDeep({extra: 'additional data'});
   });
 
+  it('enables strict delete by default', async () => {
+    const p = await repo.create({slug: 'pencil'});
+    await expect(repo.deleteById(10000)).to.be.rejectedWith(
+      /No instance with id/,
+    );
+  });
+
+  it('disables strict delete via configuration', async () => {
+    @model({settings: {strictDelete: false}})
+    class Pencil extends Entity {
+      @property({id: true})
+      id: number;
+      @property({type: 'string'})
+      name: string;
+    }
+
+    const pencilRepo = new DefaultCrudRepository<
+      Pencil,
+      typeof Pencil.prototype.id
+    >(Pencil, new DataSource({connector: 'memory'}));
+
+    await pencilRepo.create({
+      name: 'Green Pencil',
+    });
+
+    await expect(pencilRepo.deleteById(10000)).to.be.fulfilledWith(false);
```

Comment:
Why is it `fulfilledWith(false)` ... shouldn't it be `true`?

Reply:
It is in `strictDelete: false` mode, and it cannot find id `10000`; instead of rejecting it resolves with `false`.

Reply:
> It is in strictDelete: false mode, and it cannot find id 10000; instead of rejecting it resolves with false.

Could you please capture this information in a code comment?

I am expecting more readers to be confused about `.fulfilledWith(false`).